cli/src/config.rs

```diff
@@ -305,6 +305,7 @@ impl Default for RegistryConfig {
 pub struct ProviderConfig {
     pub cluster: Cluster,
     pub wallet: WalletPath,
+    pub skip_lint: Option<bool>,
```

Comment:
Let's move this to the features section above.

Reply:
Done, and I switched to use serde(default) instead of Option to be consistent with the other options.